### PR TITLE
[4.0] Mobile template toolbar

### PR DESCRIPTION
**Pull request by brianteeman**
This PR contains the markup for a proposed change to the display of the toggled toolbar.

~It is does NOT contain the css changes that are required. I really could do with some help with converting the css below to appropriate scss - or even better scss~

~You can paste the css below into your browser dev tools after applying the PR to see the intended behaviour~

~Pull Requests welcome to create the scss~
Pull Request for Issue #26486

### CODE REVIEW

**Comment by Fedik** [comment]
@brianteeman place this to `/administrator/templates/atum/scss/blocks/_toolbar.scss` at bottom and run `node build.js --compile-css`

```css
@include media-breakpoint-down(xs) {
  .toggler-toolbar {
    top: 20px;

    .toggler-toolbar-icon::before {
      font: normal normal normal 30px/1 'Font Awesome 5 Free';
      content: "\f00d";
      color: var(--toggle-color);
    }

    &.collapsed .toggler-toolbar-icon::before {
      content: "\f085";
    }
  }

  .subhead{

    joomla-toolbar-button,
    .btn-group,
    .btn{
      width: 100%;
    }

    .btn{
      [class^='icon-'],
      [class*=' icon-'],
      [class^='fa-'],
      [class*=' fa-'] {
        float: left;
      }
    }
  }
}
```

**Comment by brianteeman** [comment]
Thanks @Fedik

**Comment by snehaM26** [comment]
I have tested this item :red_circle: unsuccessfully on 98a5678d7e953d0d3c1d53dbe4f626ab2d31fc63<br /><br />Tested unsuccessfully<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26513">issues.joomla.org/tracker/joomla-cms/26513</a>.</sub>

**Comment by nidhi1709** [comment]
Tested successfully

**Comment by nidhi1709** [comment]
I have tested this item :white_check_mark: successfully on 98a5678d7e953d0d3c1d53dbe4f626ab2d31fc63<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26513">issues.joomla.org/tracker/joomla-cms/26513</a>.</sub>

**Comment by snehaM26** [comment]
I have tested this item :white_check_mark: successfully on 98a5678d7e953d0d3c1d53dbe4f626ab2d31fc63<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26513">issues.joomla.org/tracker/joomla-cms/26513</a>.</sub>

**Comment by snehaM26** [comment]
I have tested this item :white_check_mark: successfully on 98a5678d7e953d0d3c1d53dbe4f626ab2d31fc63<br /><br />Tested successfully.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26513">issues.joomla.org/tracker/joomla-cms/26513</a>.</sub>

File: administrator/templates/atum/scss/blocks/_toolbar.scss [hunk 1]
```diff
@@ -378,13 +378,47 @@ joomla-toolbar-button {
...
+    &.collapsed .toggler-toolbar-icon::before {
+      content: "\f085";
+    }
+  }
+
+  .subhead{
+
+    joomla-toolbar-button,
+    .btn-group,
+    .btn{
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_toolbar.scss]:
```suggestion
    .btn {
```

File: administrator/templates/atum/scss/blocks/_toolbar.scss [hunk 2]
```diff
@@ -378,13 +378,47 @@ joomla-toolbar-button {
...
+      width: 100%;
+    }
+
+    .btn{
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_toolbar.scss]:
```suggestion
    .btn {
```

File: administrator/templates/atum/scss/blocks/_toolbar.scss [hunk 3]
```diff
@@ -378,13 +378,47 @@ joomla-toolbar-button {
...
+      content: "\f00d";
+      color: var(--toggle-color);
+    }
+
+    &.collapsed .toggler-toolbar-icon::before {
+      content: "\f085";
+    }
+  }
+
+  .subhead{
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_toolbar.scss]:
```suggestion
  .subhead {
```

**Comment by brianteeman** [comment]
Codestyle changes applied - thanks@quy

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26513">issues.joomla.org/tracker/joomla-cms/26513</a>.</sub>

**Comment by HLeithner** [comment]
Thanks.

**Comment by infograf768** [comment]
Totally dead in RTL where the icons buttons are misplaced (tested after also using PR https://github.com/joomla/joomla-cms/pull/26613 which also corrects the place of the toggler)


Screen Shot 2019-10-19 at 10 52 12

**Comment by infograf768** [comment]
Making patch for RTL now.

**Pull request merged**